luigi/configuration.py

```diff
@@ -33,11 +33,12 @@
 import os
 import warnings
 
-from configparser import Interpolation
 try:
     from ConfigParser import ConfigParser, NoOptionError, NoSectionError
+    Interpolation = object
```

Comment:
this fixed an error i was getting when i tried to test it on an ODIN. i don't know how this wasn't a problem in production since `configparser` is only available in python3. since `Interpolation` isn't in `ConfigParser`, it gets set to `object`. I pulled this code snippet from the upstream Luigi repo